torch/jit/__init__.py

```diff
@@ -868,7 +868,7 @@ def _get_methods(cls):
 _compiled_methods_whitelist = {
     'forward', 'register_buffer', 'register_parameter', 'add_module',
     '_apply', 'apply', 'cuda', 'cpu', 'type', 'float', 'double', 'half',
-    'state_dict', 'load_state_dict', '_load_from_state_dict',
+    'state_dict', 'load_state_dict', 'diff_state_dict', '_load_from_state_dict',
```

Comment:
@zou3519 does the JIT know how to compile functions using python `set`?

Reply:
What do you mean by "compile functions using python set"?

Reply:
I probably don't know what I'm saying, but what's the purpose of `_compiled_methods_whitelist` in the JIT?